samples/openthread/coap_client/src/ble_utils.c

```diff
@@ -143,8 +154,18 @@ static void pairing_failed(struct bt_conn *conn, enum bt_security_err reason)
 	LOG_INF("Pairing failed conn: %s, reason %d", log_strdup(addr), reason);
 }
 
-int ble_utils_init(struct bt_nus_cb *nus_clbs,
-		   ble_connection_cb_t on_connect,
+int software_update_confirmation_handler(uint32_t offset, uint32_t size,
```

Comment:
I think this function should be static and enabled conditionally - same as it use.